### DISCUSSION TITLE
Propuesta del diseño contextual que dió inicio al proyecto.

### DISCUSSION THREAD

**Discussion by NeftaliYagua** [Coderic/ProtejoApp#9] [Ideas]
### Características 
- [ ] Multiidioma

 ### Opciones
- [ ] Me gusta
- [ ] Compartir
- [ ] Comentarios
- [ ] Mensajeria

 ### Redes
- [ ] Veterinarias
- [ ] Agropecuaria
- [ ] Tiendas de accesorios
- [ ] Ropa para mascotas
- [ ] Doctores y Especialistas
- [ ] Peluquerias
- [ ] Escuelas

**Comment by Richi-Mi** [comment]
Tengo algunas preguntas acerca de la app.

**En cuantos idiomas debe estar disponible la primera versión de la app?**

**Cuáles son los requerimientos funcionales iniciales de la primera versión de la aplicación?**
